libkbfs/block_retrieval_queue.go

```diff
@@ -66,6 +69,14 @@ type blockPtrLookup struct {
 	t  reflect.Type
 }
 
+// blockRetriever specifies a method for retrieving blocks asynchronously
+type blockRetriever interface {
```

Comment:
The prefetcher only needs one method and I didn't want to pass in the whole queue pointer. It's a convenient way of separating concerns.